File: id_DNA/AH/001_AH_CountingDNANucleotides.pl
#!/usr/bin/perl

use strict;
use warnings;

my $string="AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC";

my $Acount = () = $string =~ /A/g;
my $Ccount = () = $string =~ /C/g;
my $Gcount = () = $string =~ /G/g;
my $Tcount = () = $string =~ /T/g;

print "$Acount $Ccount $Gcount $Tcount\n";

#my @Cs = $string =~ /C/g;
#print join(", ", @Cs)."\n";
#my $Ccout2 = (@Cs);
#print $Ccout2."\n";

File: id_DNA/AH/001_AH_CountingDNANucleotides.py
#!/usr/bin/env python 2

string="AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC";

import re

print "%s %s %s %s" %(len(re.findall("A", string)), len(re.findall("C", string)),len(re.findall("G", string)),len(re.findall("T", string)))
